docs/source/api/pyresample.bilinear.rst

```diff
@@ -8,15 +8,15 @@ pyresample.bilinear.xarr module
 -------------------------------
 
 .. automodule:: pyresample.bilinear.xarr
-   :members:
-   :undoc-members:
-   :show-inheritance:
+    :members:
```

Comment:
Did you manually move these over one space? These were all generated by sphinx and look correct the way I had originally had them (especially since sphinx-apidoc did them)?

Reply:
Hei David, `sphinx-apidoc 1.8.5` did it automatically for me. I now see that 1.8.5 is maybe old, so I'll install a newer one and see what happens.

Reply:
Yes, this was the old version of Sphinx that used 4 spaces.

I have now run `sphinx 3.0.3` and we are back to 3-space indenting. https://github.com/pytroll/pyresample/pull/271/commits/53911766d4a9ff3fb938a12c8840c9e70db86f04